src/cascade_at/dismod/api/fill_extract_helpers/utils.py

```diff
@@ -1,6 +1,18 @@
 import pandas as pd
 
 
+def vec_to_midpoint(array):
+    """
+    Computes the midpoint between elements in an array.
+    Args:
+        array: (np.array)
+
+    Returns: (np.array)
+
+    """
+    return (array[1:] + array[:-1]) / 2
```

Comment:
I think this is more a question than a recommendation... We are fine with this returning a float in the case that the array has an odd number of elements?